config/development.json

```diff
@@ -11,5 +11,8 @@
 	},
 	"learn": {
 		"max_static_paths": 1
+	},
+	"flags": {
+		"enable_hcp_vault_secrets_api_docs_revision": false
```

Comment:
Did you want this to be true for local dev / previews?

Reply:
Soon I think! Like you mentioned, was thinking I'd flip these once the placeholder is more meaningful (so that if anyone wants to work on HCP Vault Secrets content in the meantime, they can still see changes in deploy previews!)